local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -745,7 +761,7 @@ void LocalPlannerNode::publishGround() {
 }
 
 void LocalPlannerNode::printPointInfo(double x, double y, double z) {
-  geometry_msgs::PoseStamped drone_pos = local_planner_.getPosition();
+  geometry_msgs::PoseStamped drone_pos = local_planner_->getPosition();
   int beta_z = floor(
```

Comment:
isn't there a function in common.cpp for this?

Reply:
I'd rather not change that in this PR